core/sr-primitives/src/traits.rs

```diff
@@ -124,6 +124,182 @@ pub trait BlockNumberToHash {
 	}
 }
 
+/// The account with the given id was killed.
```

Comment:
this stuff doesn't really belong in core since it's quite specific to SRML. i would just lump it into `srml-support` for now under a new `traits.rs`

Reply:
yes done